SystemMicrocontroller_Sensors/Core/Src/lcd20x4.c

```diff
@@ -0,0 +1,331 @@
+/*
+ * lcd20x4.c
+ *
+ *  Created on: May 15, 2021
+ *      Author: jdrqu
+ */
+
+#include "lcd20x4.h"
+
+const uint8_t ROW_20[] = {0x00, 0x40, 0x14, 0x54}; //address of the first position of each row for a 20x4 LCD
+
+
+void LCD_Init(LCD_HandleTypeDef * lcd){
+
+	LCD_PinInit(lcd);
+	DELAY(5);
+	if(lcd->Mode == LCD_MODE_4BITS)
```

Comment:
Please when doing comparations write macro first because of defensive programming. 
(i.e 0 == var )